vendor/curve25519-dalek-derive/.gitignore

```diff
@@ -0,0 +1,2 @@
+/target
```

Comment:
Do we really need a `vendor/` directory?

I'd just put this in a toplevel directory called `curve25519-dalek-derive`, and perhaps eventually we can move the rest of the files in the toplevel directory into `curve25519-dalek`

Reply:
yeah sure, it was already there so I thought to follow that. will move.